generators/app/templates/tests/features/test.config.js

```diff
@@ -0,0 +1,50 @@
+const path = require('path');
```

Comment:
I'd like to explore seeing if this test config can extend off of a base webpack config. I think this will help reduce duplication in this file and the core webpack config file.